### PR TITLE
Incorporated my action item and other edits

### PR DESCRIPTION
**Pull request by aphillips**
This PR contains the merge of Richard's previous work, plus a series of edits I made to the section on characters.

***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/34.html" title="Last updated on Sep 12, 2019, 3:58 AM UTC (cca14b2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/34/2e6ac07...aphillips:cca14b2.html" title="Last updated on Sep 12, 2019, 3:58 AM UTC (cca14b2)">Diff</a>

### CODE REVIEW

**Comment by aphillips** [comment]
I purposefully did not merge this myself. @r12a could you have a look?

File: index.html [hunk 1]
```diff
@@ -193,11 +193,15 @@ <h3>Defining language values</h3>
  
     <p class="advisement" id="lang_values_valid"><a class="self" href="#lang_values_valid">&#x200B;</a>Be specific about what level of conformance you expect for language tags. The word "valid" has special meaning in BCP 47. Generally "well-formed" is a better choice.</p>
  
- <p>A well-formed BCP 47 language tag has hyphen-separated subtags with specific lengths and in a particular order. Valid BCP 47 language tags are well-formed but also use only subtags that are listed in the IANA Subtag Registry. Note that the IANA Subtag Registry is frequently updated with new subtags.</p>
+ <p>A <strong>well-formed</strong> BCP 47 language tag has hyphen-separated subtags with specific lengths and in a particular order. A <strong>valid</strong> BCP 47 language tag is well-formed but also is ensures that only subtags that are listed in the IANA Subtag Registry are used. Note that the IANA Subtag Registry is frequently updated with new subtags.</p>
```
Comment (r12a) [inline index.html]: is ensures -> it ensures

File: index.html [hunk 2]
```diff
@@ -193,11 +193,15 @@ <h3>Defining language values</h3>
...
+ 
+ <p>When specifying conformance, <q>well-formed</q> is usually preferred because the validity requirements are more difficult to manage and don't add value to the specification in question; in most cases, specifications are primarily concerned with matching tags or ranges of tags rather than the meaning of each subtag.The subtag registry is fairly sizeable and is updated with new subtags and other information sporadically throughout the year. Validation of the contents of the subtags is not required for this.</p>
```
Comment (r12a) [inline index.html]: subtag.The -> subtag. The

Comment (r12a) [inline index.html]: in "is not required for this", "for this" is a bit vague.

File: index.html (hunk 1 above)
Comment (aphillips) [reply index.html]: addressed

File: index.html (hunk 2 above)
Comment (aphillips) [reply index.html]: fixed

Comment (aphillips) [reply index.html]: fixed in rewrite

**Comment by r12a** [comment]
I think we should limit this PR to just the language-specific issues (plus perhaps any typos or very small editorial fixes) and untangle it from the other edits related to characters.  The stuff in the characters section needs a lot more review and i can see a number of issues - not least that we created an article to replace much of the text that is being proposed here.

**Comment by aphillips** [comment]
> 
> 
> I think we should limit this PR to just the language-specific issues (plus perhaps any typos or very small editorial fixes) and untangle it from the other edits related to characters. The stuff in the characters section needs a lot more review and i can see a number of issues - not least that we created an article to replace much of the text that is being proposed here.

Agreed. 

I forgot that I had an open PR before I started doing my edits. When I (eventually) did a pull of changes from w3c to my private repo and did the merge, I mistakenly merged the character changes. That change needs to be removed. In our private email conversation, I noted that I hadn't created a PR (because I noticed that I needed to make the change) but apparently had this PR already open. I will remove the character changes and we should be good to merge. (The only other change should be the respec fix)

**Pull request closed without merging**